external-site/index.html

```diff
@@ -0,0 +1,88 @@
+{% extends "layout-full.html"  %}
+{% set request_host = 'http://' + request.host -%}
+{% set request_referrer = request.referrer or request_host -%}
+
+{% set request_url_param = request.args.get('ext_url') or '' -%}
+{% set external_url = request_url_param or request_host %}
+
+{# Turning this feature off until sanitized_domains is implemented.
+	{% import "_vars-external-site.html" as vars with context %}
+
+	// Determine if url param is in sanitized domains.
+	{% set external_dict = {'is_external': false} %}
+	{% set _ignore = external_dict.update({ 'is_external' : true })
+	                 if ( request_url_param.find('www.' + domain)  > -1 )
+	                 or ( request_url_param.find('//' + domain)  > -1 )
+	 %}
+	{% endfor %}
+	{% set external_url = (request_url_param if external_dict.is_external)
+	                      or request_host
+	%}
+#}
+
+{% block title -%}
+	External Site
+{%- endblock %}
+
+{% block desc -%}
+   	External Site
+{%- endblock %}
+
+{% block header -%}
+	</div>
+	<div class='sliding-nav_page'>
+	<header class="header
+	               wrapper
+	               wrapper__match-content"
+	        role="banner">
+       	<a href="/">
+            <img class="header_logo u-js-only"
+                 src="/static/img/logo_sm-exec.svg"
```

Comment:
Confirm that necessary content is accessible in IE8 http://caniuse.com/#feat=svg-img